app/api/events.py

```diff
@@ -210,15 +278,33 @@ def before_get_object(self, view_kwargs):
                 else:
                     view_kwargs['id'] = None
 
-    decorators = (is_coorganizer, )
+    def after_get(self, result):
+        result = changeToTimezone(result)
```

Comment:
@niranjan94 @shubham-padia 
I have been looking into this and found [this in marshmallow fields](http://marshmallow.readthedocs.io/en/latest/api_reference.html#marshmallow.fields.DateTime).

It says 
>Timezone-naive datetime objects are converted to UTC (+00:00) by Schema.dump. Schema.load returns datetime objects that are timezone-aware.

But when I am sending 
```
			"starts-at":"2017-07-12T11:22:33",
			"ends-at":"2017-07-16T11:22:33",
```
I get output as 
```
            "starts-at": "2017-07-12T11:22:33+00:00",
            "ends-at": "2017-07-16T11:22:33+00:00",
```

Which is as per the docs , but it is considering this 
```
			"starts-at":"2017-07-12T11:22:33+05:30",
			"ends-at":"2017-07-16T11:22:33+05:30",
```
 also as **timezone-naive** and converting it to UTC .

I checked whether the object is being created with `tzinfo` or not so I dumped the data `after_create_object` and the dump was
 `'ends_at': datetime.datetime(2017, 7, 16, 11, 22, 33, tzinfo=tzoffset(None, 19800)),`

So, object is creating alright but it is not being saved correctly.
Any suggestions ?